x/foundation/foundation.go

```diff
@@ -286,10 +286,6 @@ func (p ThresholdDecisionPolicy) ValidateBasic() error {
 }
 
 func (p ThresholdDecisionPolicy) Validate(config Config) error {
-	if p.Threshold.LT(config.MinThreshold) {
-		return sdkerrors.ErrInvalidRequest.Wrap("threshold must be greater than or equal to min_threshold")
-	}
```

Comment:
To remove threadhold checking with Minthreadhold is OK. But I think `Threadold` should be at least greater then 1. Is it right? It then, I think checking is need.

Reply:
The logic is in `ValidateBasic()` of it.